lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -489,8 +508,9 @@ def test_weighted_sum_with_unknown_units_lazy_y(self):
 class Test_collapsed__multidim_weighted_with_cube(
     Test_collapsed__multidim_weighted_with_arr
 ):
-    def setUp(self):
-        super().setUp()
+    @pytest.fixture(autouse=True)
+    def _setup(self):
+        super()._setup()
```

Comment:
You're not supposed to call fixtures directly. If you need to inherit some behaviour then I would give this fixture a new name like `_setup_subclass` and then pass `_setup` as an argument (I have some examples in my PRs)

Reply:
Had this idea on Friday:

```python
@pytest.fixture(autouse=True)
def _setup(self, ParentClass._setup):
   ...
```